dataprofiler/profilers/unstructured_text_profile.py

```diff
@@ -0,0 +1,223 @@
+        :param other2: second profile
+        :type other1: BaseColumnProfiler
+        :type other2: BaseColumnProfiler
+        """
+        if other1.name == other2.name:
+            self.name = other1.name
+        else:
+            raise ValueError("Text names unmatched: {} != {}"
+                             .format(other1.name, other2.name))
+
+        self.times = defaultdict(
+            float, {key: (other1.times.get(key, 0)
+                          + other2.times.get(key, 0)
+                          + self.times.get(key, 0))
+                    for key in (set(other1.times) | set(other2.times)
+                                | set(self.times))}
+        )
+
+        self.sample_size = other1.sample_size + other2.sample_size
+
+    def __add__(self, other):
+        """
+        Merges the properties of two TextProfiler profiles
+
+        :param self: first profile
+        :param other: second profile
+        :type self: TextProfiler
+        :type other: TextProfiler
+        :return: New TextProfiler merged profile
+        """
+        if not isinstance(other, TextProfiler):
+            raise TypeError("Unsupported operand type(s) for +: "
+                            "'TextProfiler' and '{}'".format(
+                other.__class__.__name__))
+        merged_profile = TextProfiler(None)
+        
+        if self.name == other.name:
+            merged_profile.name = self.name
+        else:
+            raise ValueError("Text names unmatched: {} != {}"
+                             .format(self.name, other.name))
+
+        merged_profile.times = defaultdict(
+            float, {key: (self.times.get(key, 0)
+                          + other.times.get(key, 0))
+                    for key in (set(self.times) | set(other.times))}
+        )
+        
+        BaseColumnProfiler._merge_calculations(merged_profile.__calculations,
+                                 self.__calculations,
+                                 other.__calculations)
+
+        if "vocab" in merged_profile.__calculations:
+            merged_profile.vocab = self.vocab.copy()
+            merged_profile._update_vocab(other.vocab)
+            
+        if "words" in merged_profile.__calculations:
+            merged_profile.word_count = self.word_count.copy()
+            for word in other.word_count:
+                if word in merged_profile.word_count:
+                    merged_profile.word_count[word] += other.word_count[word]
+                elif word not in self._stop_words:
+                    merged_profile.word_count[word] = other.word_count[word]
+                    
+        merged_profile.sample_size = self.sample_size + other.sample_size
+
+        return merged_profile
+
+    @property
+    def profile(self):
+        """
+        Property for profile. Returns the profile of the column.
+
+        :return:
+        """
+        profile = dict(
+            vocab=self.vocab,
+            words=list(self.word_count.keys()),
```

Comment:
if we have word_count, do we also need words?
Should word_count be ordered by value?

Reply:
Will order word count by value. 
Words, I'm indifferent about. It was in our pseudo code so I left it